apps/vanilla/src/pages/blackjack/game/debug/debugInfo.ts

```diff
@@ -0,0 +1,32 @@
+namespace Game.Debug {
+
+	/**
+	 * Debug info.
+	 */
```

Comment:
We prefer writing single line comments this way
```suggestion
	/** Debug info. */
```

Reply:
What about it?
![image](https://github.com/user-attachments/assets/773b5ac0-03ff-4a78-b28c-44e0e02ee612)